src/components/Article.jsx

```diff
@@ -1 +1,23 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title,
+  text,
+  date,
+  dateISO = date.toISOString().slice(0, 10),
+  dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  }) }) => (
+    <div className="box">
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
```

Comment:
```suggestion
export const Article = ({ title, text, date }) => {
  const dateISO = date.toISOString().slice(0, 10),
  const dateLocaleString = date.toLocaleDateString('en-US', {
    year: 'numeric',
    month: 'long',
    day: 'numeric',
  })
  
  return (
    <div className="box">
      <h2 className="title" data-cy="title">
        {title}
      </h2>
```


it would be better to create a separate variables for `dateISO` and `dateLocaleString`, not passing them in props